src/test/Assets/TestProjects/StandaloneApp/StandaloneApp.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <AssemblyName>StandaloneApp</AssemblyName>
+    <OutputType>Exe</OutputType>
+    <PackageId>StandaloneApp</PackageId>
+    <RuntimeIdentifiers>$(TestTargetRid)</RuntimeIdentifiers>
```

Comment:
If you are setting `RuntimeIdentifier`, you don't need to set `RuntimeIdentifiers`.  Just like TargetFrameworks, the plural and the singular shouldn't be used at the same time.